src/cap-description.adoc

```diff
@@ -7,16 +7,70 @@ space. There are two primary ISA variants, RV32I and RV64I, which provide
 32-bit and 64-bit address spaces respectively. The term XLEN refers to the
 width of an integer register in bits (either 32 or 64). The value of XLEN may
 change dynamically at run-time depending on the values written to CSRs, so we
-define XLENMAX to be widest XLEN that the implementation supports.
-
-
-{cheri_base_ext_name} defines capabilities of size CLEN corresponding to 2 *
-XLENMAX without including the tag bit. The value of CLEN is always calculated
-based on XLENMAX regardless of the effective XLEN value.
+define *XLENMAX* to be widest XLEN that the implementation supports.
+
+{cheri_base_ext_name} defines capabilities which act on virtual addresses of
```

Comment:
```suggestion
{cheri_base_ext_name} defines capabilities which act on addresses of
```
I'd drop the virtual here since we also want to support systems without virtual memory.